examples/stub_generation/README.md

```diff
@@ -12,7 +12,7 @@ generator. This particular example defines a Kernel with four fields,
 To run this example type:
 
 ```sh
-python ../../src/psyclone/gen_kernel_stub.py testkern_stencil_multi_mod.f90
+genkernelstub testkern_stencil_multi_mod.f90
 ```
```

Comment:
This example does not run for me, it can't find the python config file. Afaik genkernelstub does not have  a --config option, so you might have to set PSYCLONE_CONFIG. This is not caused by this PR, but I think worth fixing.

Reply:
Setting PSYCLONE_CONFIG does fix it so I've done that. Is that sufficient or do you want me to open an issue about kernel-stub generator command-line flags?

Reply:
I am happy with the environment variables. If it would be useful for others to have a command line option, they can open a ticket imho.